scripts/max-loader.js

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env node
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const async = require('async');
+const fs = require('fs');
+const path = require('path');
+const request = require('request');
+const requestProgress = require('request-progress');
+const mkdirp = require('mkdirp');
+const cp = require('child_process');
+const CronJob = require('cron').CronJob;
```

Comment:
we might have to remove new stuff such as `const` because we might still be on node 0.10 when this is ready.